phpcs.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?>
```

Comment:
This files needs to be reworked, as it should now use our new coding standard. Similar to https://github.com/matomo-org/matomo/blob/5.x-dev/phpcs.xml, but you need to adjust it for the plugin accordingly.

Reply:
Done